_data/site_menu.yml

```diff
@@ -84,7 +84,27 @@ using:
             path: "/docs/reference/latest/managing_providers/configuration_management_providers/red_hat_satellite_6.html"
             desc: "Managing Red Hat Satellite 6 Providers"
 
-        - title: Automation Management Providers
+        - title: Automation Providers
+          children:
+          - title: Ansible Tower
+            path: "/docs/reference/latest/managing_providers/automation_management_providers/ansible_tower.html"
+            desc: "Managing Ansible Tower Providers"
```

Comment:
Should this be Ansible Automation Platform now?

Reply:
Also do we have AWX?  Or perhaps this should be Ansible Automation Platform / AWX?

Reply:
Yeah we can definitely do that, I was avoiding moving&changing in the same PR but that makes sense

Reply:
Agreed - let's make it a different PR